community/index.md

```diff
@@ -0,0 +1,12 @@
+---
+sidebar_position: 1
+---
+
+# About conda-forge
```

Comment:
I'd use the Docusaurus card list components here.

Reply:
Getting an error about not having cattegories for the community page